### DISCUSSION TITLE
How to ignore the redundant export?

### DISCUSSION THREAD

**Discussion by Gehbt** [posva/unplugin-vue-router#288] [Questions and Help]
When I type one letter such as 'd', there have some bundled exports.🤔
![image](https://github.com/posva/unplugin-vue-router/assets/74761884/5e6d8ef8-2291-4fb1-96b2-debe3246cac1)
It is redundant, and how to overlook it?🤔

**Comment by posva** [comment]
I think the package.json is misconfigured. Probably due to how the `client.d.ts` is exposed 🤔 

One way to get around this is to create a `.vscode/settings.json` file and add

```json
{
  "typescript.preferences.autoImportFileExcludePatterns": [
    "vue-router",
    "unplugin-vue-router/dist"
  ]
}
```

**Reply by Gehbt** [reply]
It is useful in Linux(Dev Container), but fail in Windows
